dist/index.js

```diff
@@ -4788,7 +4789,7 @@ async function getElixirVersion(exSpec0, otpVersion) {
   const elixirVersions = await getElixirVersions()
   const semverVersions = Array.from(elixirVersions.keys()).sort()
 
-  const exSpec = exSpec0.match(/^(.+)(-otp-.+)/) || exSpec0.match(/^(.+)/)
+  const exSpec = exSpec0.match(/^v?(.+)(-otp-.+)/) || exSpec0.match(/^v?(.+)/)
```

Comment:
We remove the `v` from the input, to later add it back again (this way we allow `v1.12` as well as `1.12`).